### ISSUE TITLE
Can't instantiate Geoparser

### ISSUE THREAD

**Issue by RomainMularczyk** [openeventdata/mordecai#80]
Hi, it seems I can't instantiate Geoparser properly.

Here is the errors Python returns :

![error](https://user-images.githubusercontent.com/54244080/86924735-274a6780-c130-11ea-8b45-53fa51f49dc4.PNG)

Thank you !

**Comment by zacharyblank** [comment]
Same here!

**Comment by ahalterman** [comment]
This seems to be causes by a mismatch between the older version of Keras used to train the models and the updated tensorflow/keras version. I've retrained the models using updated versions of all the libraries: https://github.com/openeventdata/mordecai/tree/retrain. Does that branch fix the issue for you? Note that I also refreshed the Geonames index, so you'll also have to re-download the Geonames index from [here](https://andrewhalterman.com/files/geonames_index.tar.gz).
